webapp/packages/core-blocks/src/FormControls/Combobox.tsx

```diff
@@ -169,6 +171,19 @@ export const Combobox: ComboboxType = observer(function Combobox({
     }
   }
 
+  function handleClear() {
+    setInputValue(null);
+    if (name !== undefined && state) {
+      state[name] = undefined;
+    }
+    if (onSelect) {
+      onSelect(undefined as any, name as undefined, selectedKey);
+    }
+    if (context) {
+      context.change('', name);
+    }
+  }
```

Comment:
Why do we need manual handling? Don't the already-provided callbacks cover it for the checkbox itself?